openapi-spec/swagger.yaml

```diff
@@ -679,6 +679,76 @@ paths:
             application/xml:
               schema:
                 $ref: '#/components/schemas/BaseResponse'
+  '/v1/s3/{bucketName}/?lifecycle':
```

Comment:
I don't understand why there is a `?` here.

Reply:
That lifecycle is a query parameter, to show that we have ?, I will discuss if not needed then will remove it